src/eShop.AppHost/Program.cs

```diff
@@ -65,44 +65,45 @@
     .WithReference(rabbitMq)
     .WithEnvironment("IdentityUrl", idpHttps);
 
-#if USE_AZURE_OPENAI
-
-const string openAIName = "openai";
-
-// to configure an existing OpenAI connection, set it in the ApHost's user secrets
-IResourceBuilder<IResourceWithConnectionString> openAI;
-string textEmbeddingName = "text-embedding-ada-002";
-string chatModelName = "gpt-35-turbo-16k";
-if (builder.Configuration.GetConnectionString(openAIName) is not null)
+// set to true if you want to use OpenAI
+bool useOpenAI = false;
+if (useOpenAI)
 {
-    openAI = builder.AddConnectionString(openAIName);
-    if (builder.Configuration["AI:OpenAI:EmbeddingName"] is string embedName)
+    const string openAIName = "openai";
+    const string textEmbeddingName = "text-embedding-ada-002";
+    const string chatModelName = "gpt-35-turbo-16k";
+
+    // to use an existing OpenAI resource, add the following to the AppHost user secrets:
```

Comment:
Remove this commented out code.

Reply:
This isn't "commented out code". it is the JSON you need to put in your user secrets to enable open AI in your app.